arcade-services.sln

```diff
@@ -90,9 +90,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ProductConstructionService.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tools.Common", "tools\Tools.Common\Tools.Common.csproj", "{977060F4-EEEF-420D-02E8-5E58D3E51942}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VersionPropsFormatter", "tools\VersionPropsFormatter\VersionPropsFormatter.csproj", "{40B3EEE3-2B38-4264-A32F-173D129090F8}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VersionPropsFormatter.Tests", "test\VersionPropsFormatter.Tests\VersionPropsFormatter.Tests.csproj", "{9B43D741-AED1-42DA-88EA-19CB1D71C41C}"
```

Comment:
Shouldn't this project stay?

Reply:
The tests are already kind of covered in the merge policy tests, so I deleted them